moveos/moveos-stdlib/moveos-stdlib/sources/account_storage.move

```diff
@@ -55,6 +55,12 @@ module moveos_std::account_storage {
         object_storage::contains<AccountStorage>(object_storage, object_id)
     }
 
+    public fun ensure_account_storage(ctx: &mut StorageContext, account: address) {
+        if (!exist_account_storage(ctx, account)) {
```

Comment:
account storage will create when account create, so after account created, there is no scenario where account storage does not exist, without considering the test scenario